src/App.css

```diff
@@ -1,38 +1,261 @@
+
+html { box-sizing: border-box; font-size:125%;}
+
+*, *:before, *:after {box-sizing: inherit}
+
+.sr-only { position: absolute; width: 1px; height: 1px; margin: -1px; border: 0; padding: 0; white-space: nowrap; clip-path: inset(100%); clip: rect(0 0 0 0); overflow: hidden;}
+img { width: 100%; display: block; height: auto;}
+
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@400;700&display=swap");
+@import url("https://fonts.cdnfonts.com/css/pokemon-solid");
+
+*{
+    padding: 0;
+    margin: 0;
+}
+
+body{
+    background-color: rgb(213, 0, 0);
+    font-family: 'Open Sans', sans-serif;
+    color: yellow;
+    text-shadow: 5px 5px 0px #00077C;
+}
+
+.title{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-family: 'Pokemon Solid', sans-serif;
+    color: yellow;
+    text-shadow: 5px 5px 0px #00077C;
+    font-size: 2rem;
+    flex-direction: column;
+}
+
+.title form{
+    font-size: 1rem;
+}
+
+.container{
+    width: 90%;
+    margin: auto;
+    padding-top: 50px;
+    display: flex;
 }
 
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.left-content,
+.right-content{
+    flex-basis: 50%;
 }
 
-@media (prefers-reduced-motion: no-preference) {
-  .App-logo {
-    animation: App-logo-spin infinite 20s linear;
+.left-content{
```

Comment:
Its not 100% clear that you can click on the boxes on the left, maybe changing the cursor to pointer so the use knows that they can click on them!